gio/src/application.rs

```diff
@@ -25,6 +26,8 @@ pub trait ApplicationExtManual: IsA<Application> {
         };
         ExitCode::from(exit_code)
     }
+
+    #[doc(alias = "open")]
```

Comment:
I added this alias on `connect_open` and some spacing between functions as a little cleanup.